analyses/molecular-subtyping-MB/01-filter-and-batch-correction.R

```diff
@@ -42,10 +47,15 @@ polya <- readRDS(polya.file)
 stranded <- readRDS(stranded.file)
 
 # read and subset clinical file to MB samples
-clin <- read.delim(clin.file, stringsAsFactors = F)
+clin <- read.delim(clin_file, stringsAsFactors = F)
 clin.mb  <- clin %>%
-  filter(experimental_strategy == "RNA-Seq",
-         integrated_diagnosis == "Medulloblastoma") %>%
+  # Inclusion on the basis of strings in pathology_diagnosis and 
+  # pathology_free_text_diagnosis
+  filter(str_detect(str_to_lower(clin$pathology_diagnosis), 
+                    paste0(path_dx_list$include_path_dx, collapse = "|")) | 
+           str_detect(str_to_lower(clin$pathology_free_text_diagnosis), 
+                      paste0(path_dx_list$include_free_text, collapse = "|")),
```

Comment:
This doesn't appear to be doing exact matching per https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/787#issuecomment-696422415. It doesn't surprise me that this is giving us the expected result, but I think since we have the ability to be more stringent here (compared to what shows up in `pathology_diagnosis` for the HGAT samples) we can go ahead and do that.